pytorch_lightning/plugins/training_type/single_device.py

```diff
@@ -23,6 +23,8 @@ class SingleDevicePlugin(TrainingTypePlugin):
     def __init__(self, device: torch.device):
         super().__init__()
         self.device: torch.device = device
+        self.world_size = 1
+        self.local_rank = 0
```

Comment:
is this required? where is it used?

Reply:
Not in this PR, but the one to skip warning. Let me remove it.